pdns/doh.hh

```diff
@@ -247,6 +250,7 @@ struct DOHUnit
   st_h2o_req_t* req{nullptr};
   DOHUnit** self{nullptr};
   DOHServerConfig* dsc{nullptr};
+  pdns::channel::Sender<DOHUnit, void(*)(DOHUnit*)>* responseSender{nullptr};
```

Comment:
Unfortunately it wouldn't, because we have a non-static `release()` member function and a static `release(DOHUnit*)` one, so the compiler cannot decide which one we are referring to.